packages/react-components/src/organisms/EditModal.tsx

```diff
@@ -56,14 +56,14 @@ const EditModal: React.FC<EditModalProps> = ({
   }, [status, dirty]);
 
   const historyPush = usePushFromHere();
-  const asyncOnSave = async () => {
+
+  const asyncFunctionWrapper = async (cb: () => void | Promise<void>) => {
```

Comment:
```suggestion
  const asyncFunctionWrapper = async (cb: () => Promise<void>) => {
```

Should this function be returning a Promise only?

@csbarbosa  @jhonny-m